Common/Servers/FitsWriter/src/EngineThread.cpp

```diff
@@ -525,7 +527,8 @@ void CEngineThread::runLoop()
 			// create the file and save main headers
 			m_data->getFileName(fileName,filePath);
 			if (!IRA::CIRATools::directoryExists(filePath)) {
-				if (!IRA::CIRATools::makeDirectory(filePath)) {
+				IRA::CIRATools::makeDirectory(filePath);
+				if (!IRA::CIRATools::directoryExists(filePath)) {
```

Comment:
@aorlati I changed this behavior so that the exception is thrown only if the directory was not created correctly. For some reason the makeDirectory returns false even if the directory was created. The exception is not blocking, but many times the observers reported it and sometimes they even stopped observing until someone told them the exception was not an issue.